packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -19,7 +19,8 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
     constructor(@inject(WorkspaceService) wsService: WorkspaceService,
         @inject(FileSystem) protected fileSystem: FileSystem) {
         super();
-        wsService.root.then(root => {
+        wsService.workspaceRootChange.then(data => {
```

Comment:
It is minor, but maybe this could be in an `async` postConstruct method, to convert this part to attribute injection like the documentation suggests to do ?